Makefile

```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (C) 2019-2024 Intel Corporation
+
```

Comment:
From the offline discussion with @grom72 we agreed as follows:

- there are too many files in all the repos to do it for all of them right now.
- the `Makefile` file is virtually in all the repositories so it may serve as an example on how to do it right.